### ISSUE TITLE
Return new id from insert

### ISSUE THREAD

**Issue by twastvedt** [stubailo/meteor-rest#100]
I think this is a feature request, but I might just be missing something. In Meteor when I insert a new document, the function returns the `_id` of the created document. Is there a way to get that new `_id` when using meteor-rest?

**Comment by stubailo** [comment]
Hmm, I feel like this should already work. You're referring to this right? https://github.com/stubailo/meteor-rest/blob/master/packages/rest/README.md#collection-methods

Going to glance at the code.

**Comment by stubailo** [comment]
Ohhhh, I think I know what's up. If you're talking about calling `Collection.insert()` in Meteor, then that returns the ID because it's generated on the client. The underlying network call doesn't return anything.

So we could add that to this package, or you could simply generate a random ID and then pass it in as the `_id` of the posted document. Thoughts?

**Comment by romans127** [comment]
You could use rwatts:meteor-uuid for the random Id. You can generate a
small character by calling tiny or a standard iid by calling new.

**Comment by twastvedt** [comment]
@stubailo Good to know. I didn't realize that happened client side. I'm ok with either option you suggest. It would be cool if it were built in, but the alternative is quite easy. I'm not too familiar with proper REST behavior, but some Google searching suggests that responding to a POST with the created `_id` would be allowable and maybe even preferable. I'll let you decide whether that should be part of this package's domain or not though.

@rwatts3 I'm writing an Angular 2 app to communicate with my Meteor app, but it looks like I can use [meteor-random](https://www.npmjs.com/package/meteor-random) instead.

**Comment by stubailo** [comment]
Yeah, I agree responding with the ID would be great. Just writing down my realization about why it works this way now.
